### DISCUSSION TITLE
Adding South African region for deploying projects

### DISCUSSION THREAD

**Discussion by fbele** [supabase/supabase#21429] [Feature Requests]
Hi there,

Since Supabase Cloud is using AWS to deploy projects on the cloud, and AWS does have does have Data Center in South Africa (Cape Town) region available to my knowledge, can it be expected that you (the Supabase masters) will also make this region available?

Because African region in general is the only one that is not covered at all and it's kind of shame, that such a great product isn't available there.

**Comment by Steinholf** [comment]
This would really be great

**Comment by cjangrist** [comment]
+1 

 Latency from sub Sahara africa to London (usually lowest ping) is > 100ms which is a very ad user experience. 

Do you have any estimate or plan of when this will be rolled out?

**Comment by code3z** [comment]
Yes please add Africa!

**Comment by deondejongh** [comment]
+1 For adding af-south-1 / Cape Town, South Africa
Other than going directly with the big cloud providers, I'm struggling to find any solo-developer friendly services like Supabase (specifically managed-postgres) that deploys to this region. Has anyone else had luck (other than self-hosting on VPS)?

**Comment by alexbvw** [comment]
South Africa represents one of the most underserved but fastest-growing developer ecosystems in the global south. With a tech-savvy youth population, increasing remote work adoption, and a surging startup scene across fintech, healthtech, and e-commerce — developers here need fast, reliable, and affordable tools to build modern web & mobile apps.

Yet, current cloud infrastructure is either overpriced, overcomplicated, or geographically distant, resulting in higher latency and poor DX. Supabase offers the perfect balance of:

🔧 Modern Dev Stack (Postgres, auth, file storage, real-time, edge functions)

💸 Affordable Pricing for early-stage builders

🌍 Open Source Principles that resonate with South Africa’s growing open-tech movement

Expanding region-specific infrastructure and support to South Africa would empower thousands of devs to build local-first, global-ready apps — unlocking massive value in a digitally awakening continent.

**Comment by dmsapps** [comment]
Supabase, any response to this request would be appreciated 🙏

**Comment by mikesoaps** [comment]
+1 Please bring to South Africa, the latency to any other region involves crossing an entire continent!

**Comment by LuanPotg** [comment]
+1 . Love using SB as my main backend. Super capable.

**Comment by JackScottKing** [comment]
+1 Please bring this to South Africa (af south). I currently have a Supabase project hosted in Frankfurt, and the latency is around 160ms. This is affecting the overall user experience of my application. I am not just speaking for myself, this would benefit many developers across South Africa and the rest of the continent.

**Comment by waliebali77** [comment]
+1 bring to af-south-1 please!

Im thinking of self hosting on a VPS but I'm not too keen to go through the hassle of managing VPS and stuff. I guess I have no choice.

**Comment by cipherw0lf** [comment]
+1 for af-south-1

What alternatives are people using? How is firebase?

**Reply by waliebali77** [reply]
I tried self hosting Supabase but decided against it because of the complexity so now I'm using a PostgresDB hosted with Aiven on AWS af-south-1 with Drizzle ORM, and Clerk for Auth. My stack is really expensive unfortunately but it works great. If Supabase was on af-south-1 i'd use them in a heartbeat.

**Reply by waliebali77** [reply]
you can also check out AWS RDS too if you're looking for a SQL DB, I briefly looked at it and its not as extensive as Supabase but it can host a postgres/mySQL DB. Not sure how other people who have used it have found it?

**Comment by machielvdw** [comment]
+1 We need the af-south-1 region

**Comment by Lexical-Luke** [comment]
+1 Please add af-south-1 region

**Comment by Mutondi** [comment]
@kiwicopple Please help us here

**Comment by mickey-ventura** [comment]
+1 there are plenty of developers who would benefit from this. Please consider adding a region in Africa !

**Comment by geoffreygarrett** [comment]
+1 `af-south-1` region

**Comment by alexbvw** [comment]
+1 af-south-1 region

**Comment by zemuldo** [comment]
Is there a chance this is coming sometime?
DR policy for our service is forcing this.

**Reply by cipherw0lf** [reply]
I wouldn't hold my breath.

**Comment by Gyromaniak** [comment]
+1 af-south-1 region

**Comment by untilhamza** [comment]
2026 May 1st.
still nothing :(

**Comment by Sox-Dev** [comment]
+1 Please add af-south-1 region
